restapi/src/main/java/io/stargate/web/docsapi/models/DocumentResponseWrapper.java

```diff
@@ -62,13 +66,19 @@ public DocumentResponseWrapper setData(T data) {
     return this;
   }
 
+  public ExecutionProfile getProfile() {
```

Comment:
can you add the `@ApiModelProperty` here as well.. Maybe a small example would also be great!

Reply:
Annotation added